includes/admin/feedzy-rss-feeds-import.php

```diff
@@ -2272,6 +2332,40 @@ public function add_cron() {
 		if ( false === wp_next_scheduled( 'feedzy_cron' ) ) {
 			wp_schedule_event( $time, $schedule, 'feedzy_cron' );
 		}
+
+		// Register import jobs based cron jobs.
+		$import_job_crons = get_posts(
```

Comment:
In the `get_posts`, we can use the fields argument to only get the IDs to improve the performance a bit. https://developer.wordpress.org/reference/functions/get_posts/#comment-3553

I see we only use the IDs from these get_posts calls.